### PR TITLE
feat: Add mobile layout sample to storybook

### PR DESCRIPTION
**Pull request by HoonBaek**

### CODE REVIEW

File: src/modules/App/stories/index.stories.js [hunk 1]
```diff
@@ -269,6 +269,31 @@ export const Korean = () => fitPageSize(
...
+    config={{ logLevel: 'all' }}
+    replyType="THREAD"
+    isReactionEnabled
+    isMentionEnabled
+    isVoiceMessageEnabled
+    isMultipleFilesMessageEnabled
+    isMessageReceiptStatusEnabledOnChannelList
+    uikitOptions={{
+      groupChannel: {
+        // enableTypingIndicator: false,
```
Comment (AhyoungRyu) [inline src/modules/App/stories/index.stories.js:290]: Do we want to keep this comment too?

Comment (liamcho) [reply src/modules/App/stories/index.stories.js:290]: Let's just remove it. I left it there so we can test below option when enableTypingIndicator is set to false

**Review by liamcho — APPROVED** [review]
LGTM

**Comment by HoonBaek** [comment]
let me close this PR, because we are preparing the new storybook of the latest version

**Pull request closed without merging**